x-pack/plugins/apm/common/ux_ui_filter.ts

```diff
@@ -118,3 +118,9 @@ export const uxLocalUIFilters = uxLocalUIFilterNames.reduce((acc, key) => {
     },
   };
 }, {} as UxLocalUIFilterMap);
+
+export type UxUIFilters = {
```

Comment:
any chance we can move this entire file?

Reply:
this one is tricky, it's being used in APM server. And UX plugin is using some stuff from APM. So for now relationship is one directional. UX app is dependant on APM. I kept it here for now , to not introduce circular dependency. 

Once routes are removed from APM, this will be gone auto.